### ISSUE TITLE
⁠Add MongoDB (or Similar) Functionality to the API

### ISSUE THREAD

**Issue by annimukherjee** [kabaddiPy/kabaddiPy#8]
Implement support for MongoDB or a similar database system to store and query data efficiently.

**Comment by Rohxn16** [comment]
Can I get some more insight into this, I'm willing to take a look into the issue.

**Comment by annimukherjee** [comment]
Yes, currently when you `pip install` all the csvs are getting installed into the users dist-packages. 

We want to make the package light-weight as it's just a data-retriever and the starting point for analytics and analysis. Hence we wanted to migrate the data to a place where when the user pip installs, the csvs aren't installed directly but the user can still access the data.

This is the philosophy behind this issue!

**Comment by Rohxn16** [comment]
Ok, I think I get it, I will try to come up with a solution this week.

**Comment by JahnaviDhanaSri** [comment]
Hi @annimukherjee 
I understand that we’re looking to make the package lighter by migrating data storage away from direct CSV installations. I believe I can contribute effectively to this task by implementing a database solution that allows users to access data without cluttering their installations.
If you think this would be a good fit, I would love to take on this work.

**Comment by annimukherjee** [comment]
Yep! You've got it! 

Assigning this one to you @JahnaviDhanaSri

**Comment by IshuSinghSE** [comment]
Hello! I can add MongoDB to Fetch and store data instead of CSV, Could you assign this to me? @annimukherjee
